extensions/amp-form/0.1/amp-form-textarea.js

```diff
@@ -101,8 +101,7 @@ export class AmpFormTextarea {
       }
 
       const element = dev().assertElement(e.target);
-      if (element.tagName != 'TEXTAREA' ||
-          !element.hasAttribute(AMP_FORM_TEXTAREA_EXPAND_ATTR)) {
+      if (element.tagName != 'TEXTAREA') {
```

Comment:
Can we add a comment stating why we handle all `textarea` elements and not only ones with the feature attribute?

Reply:
apologies for missing this comment. I'll make the comment change in a separate PR, thanks.